src/Core/tests/Benchmarks.Droid/Benchmarks.Droid.csproj

```diff
@@ -13,8 +13,7 @@
     <RunAOTCompilation>false</RunAOTCompilation>
     <!-- Physical device is recommended -->
     <RuntimeIdentifier>android-arm64</RuntimeIdentifier>
-    <!-- HACK: to workaround System.IO.FileNotFoundException: /System.Private.CoreLib.dll -->
-    <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
```

Comment:
I had to use a nightly BenchmarkDotNet to remove this change:

https://github.com/jonathanpeppers/BenchmarkDotNet-Android/commit/2a0588793b7b05f45b71fe62d1f937ada04b9816

Did 0.13.5 work for you somehow?

We need this fix to be in the release on NuGet.org: https://github.com/dotnet/BenchmarkDotNet/commit/2cce425a69f23b2c9a0de93ceb80c0cc915e5ec6

Reply:
Hmm.. would adding the BDN prerelease feed that my team manages to the Nuget.Config file work? 
```<add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />```.
I would also update the version specified to match the one we have there (0.13.5.2136). This is also assuming that adding the nightly feed would not be doable.

Reply:
We can add a dnceng feed, but probably not one like `<add key="bdn-ci" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />`.